client/src/components/Rating/RatingStars.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faStar as EmptyStar} from '@fortawesome/free-regular-svg-icons';
+import {faStar as SolidStar}  from '@fortawesome/free-solid-svg-icons';
+import './RatingStyles.css';
+
+export const RatingStars = () => {
+  let stars = new Array(5).fill(EmptyStar);
```

Comment:
IDK why do you need mutate this variable.
You can do smth like this:
```js
const stars = new Array(5).fill(EmptyStar);

const [starSet, setStarSet] = useState(stars);
```

Reply:
I  change it here:
stars = stars.map((star, index) => {
      if(index <= rating) {
        return SolidStar; 
      } else {
        return EmptyStar;
      }
    })

Did I make it wrong?

Reply:
> Did I make it wrong?

Not wrong, but this is better to keep your functions pure. So they shouldn't mutate variables out of scope
so you can create new variable inside this function

Reply:
Ok, now I get you:
const stars = new Array(5).fill('EmptyStar'); 
const changeStars = (rating) => {
    const newStarsSet = stars.map((star, index) => {
      return index <= rating ? 'SolidStar' : 'EmptyStar'
    })
    setStarSet(newStarsSet)
  }